backend/app/services/plugins/cook_county_assessor.py

```
@@ -212,3 +222,153 @@ def _fetch_most_recent_sale(self, pin: str) -> dict:
+            plugin = CookCountyAssessorPlugin()
+            where = f"pin='{normalized}'"
+            url = (
+                _PARCEL_SALES_URL
+                + "?$select=pin,sale_date,sale_price,sale_type"
+                + "&$where=" + quote(where)
+                + "&$order=sale_date+DESC"
+                + f"&$limit={capped}"
+            )
+            try:
+                rows = plugin._cache_loader._socrata_get_with_retry(url, max_retries=2)
+            except Exception as exc:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)